Countries/CountryTableViewHeader.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ListHeader.swift
+//  Countries
+//
+//  Created by Khrystyna Matasova on 02.12.2021.
+//
+
+import Foundation
+import UIKit
+
+class CountryTableViewHeader: UITableViewHeaderFooterView {
+    
+    private enum HeaderConstants {
+        static let spacing: CGFloat = 0
+        static let stackViewIndent: CGFloat = 0
+    }
+    
+    static let identifier = "CountryTableViewHeader"
+    
+    let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.distribution = .fillEqually
+        stackView.spacing = HeaderConstants.spacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+    
+    let earthImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
line #32 is useless because you'll add `earthImageView` as arranged subview into stack